src/c++/perf_analyzer/genai-pa/genai_pa/parser.py

```diff
@@ -58,64 +81,114 @@ def add_model_args(parser):
 
 
 def add_profile_args(parser):
-    parser.add_argument(
+    profile_group = parser.add_argument_group("Profiling")
+    load_management_group = profile_group.add_mutually_exclusive_group()
+
+    profile_group.add_argument(
         "-b",
         "--batch-size",
         type=int,
         default=1,
         required=False,
-        help="The batch size / concurrency to benchmark. (Default: 1)",
+        help="The batch size to benchmark. The default value is 1.",
     )
-    parser.add_argument(
-        "--input-length",
+    load_management_group.add_argument(
+        "--concurrency",
         type=int,
-        default=128,
         required=False,
-        help="The input length (tokens) to use for benchmarking LLMs. (Default: 128)",
+        help="Sets the concurrency value to benchmark.",
     )
-    parser.add_argument(
-        "--output-length",
+    profile_group.add_argument(
+        "--max-threads",
         type=int,
-        default=128,
+        default=16,
+        required=False,
+        help="Sets the maximum number of threads that will be "
+        "created for providing desired concurrency or request rate. "
+        "The default value is 16.",
+    )
+    # TODO: necessary?
+    # parser.add_argument(
+    #     "--output-length",
+    #     type=int,
+    #     default=128,
+    #     required=False,
+    #     help="The output length (tokens) to use for benchmarking LLMs. (Default: 128)",
+    # )
```

Comment:
Assuming we'll clean up these comments blocks and unused dataset_args and stuff in later follow-up PRs